app/models/concerns/github_identity.rb

```diff
@@ -43,7 +43,7 @@ def update_repo_permissions
 
     update_column(:currently_syncing, true)
     download_orgs
-    r = github_client.repos
+    r = github_client.repos.map { |repo_data| GithubRepositoryHostDataFactory.generate_from_api(repo_data) }
```

Comment:
Think this factory got removed in [this commit](https://github.com/librariesio/libraries.io/pull/3338/commits/5f85a015837df5237fb234d7304d3cbb5848e6dc#diff-114d130d1fe1bd53c12fb4eafaeaa8d3fde39abaae7752b13c2f47cde4a2e170)...?